internal/cmd/commander.go

```diff
@@ -87,6 +87,7 @@ func SmPrepare(cmd Command, ctx *Context) func(*cobra.Command, []string) error {
 				ClientSecret:          clientConfig.ClientSecret,
 				IssuerURL:             clientConfig.IssuerURL,
 				SSLDisabled:           clientConfig.SSLDisabled,
+				UseBasicAuth:          clientConfig.TokenBasicAuth,
```

Comment:
Can we rename this to something like in the oauth package - `IsBrokenAuthHeader`?